ssz-rs/src/vector.rs

```diff
@@ -118,12 +118,13 @@ where
     T: SimpleSerialize + Default + Clone,
 {
     fn default() -> Self {
+        // TODO: statically assert that N != 0, possibly with
+        // https://crates.io/crates/static_assertions
+        assert!(N != 0);
```

Comment:
I've used `N != 0` here, but `Bitvector` uses `N > 0` - is that the preferred style?

Reply:
I think `N > 0` reads better too.

Also, I've previously tried to use `const_assert!` with const generics, and unfortunately that wasn't supported. So we can remove the `TODO`, I think.